instrumentation/opentelemetry-api-beta/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/SpanInstrumentation.java

```diff
@@ -23,25 +22,25 @@
 import net.bytebuddy.matcher.ElementMatcher;
 
 @AutoService(Instrumenter.class)
-public class DefaultSpanInstrumentation extends AbstractInstrumentation {
+public class SpanInstrumentation extends AbstractInstrumentation {
   @Override
   public ElementMatcher<? super TypeDescription> typeMatcher() {
-    return named("application.io.opentelemetry.trace.DefaultSpan");
+    return named("application.io.opentelemetry.trace.Span");
   }
 
   @Override
   public Map<? extends ElementMatcher<? super MethodDescription>, String> transformers() {
     return Collections.singletonMap(
-        isMethod().and(isPublic()).and(isStatic()).and(named("create")),
-        DefaultSpanInstrumentation.class.getName() + "$CreateAdvice");
+        isMethod().and(isPublic()).and(isStatic()).and(named("wrap")),
```

Comment:
@trask Not being able to get this instrumentation of a static interface method to kick in